Exiled.API/Features/Player.cs

```diff
@@ -1313,12 +1313,66 @@ public void Broadcast(ushort duration, string message, global::Broadcast.Broadca
         /// <param name="itemType">The item to be added.</param>
         public void AddItem(ItemType itemType) => Inventory.AddNewItem(itemType);
 
+        /// <summary>
+        /// Add the amount of items of the specified type with default durability(ammo/charge) and no mods to the player's inventory.
+        /// </summary>
+        /// <param name="itemType">The item to be added.</param>
+        /// <param name="amount">The amount of items to be added.</param>
+        public void AddItem(ItemType itemType, int amount)
+        {
+            if (amount > 0)
+            {
+                for (int i = 0; i < amount; i++)
+                    Inventory.AddNewItem(itemType);
+            }
+        }
+
+        /// <summary>
+        /// Add the list of items of the specified type with default durability(ammo/charge) and no mods to the player's inventory.
+        /// </summary>
+        /// <param name="items">The list of items to be added.</param>
+        public void AddItem(List<ItemType> items)
+        {
+            if (items.Count > 0)
+            {
+                foreach (ItemType itemType in items)
+                    Inventory.AddNewItem(itemType);
```

Comment:
```suggestion
                for (int i = 0; i < items.Count; i++)
                    AddItem(items[i]);
```

There's no reason for us to use a foreach() with variable assignment here.